conf/terraform/dev-vm.tf

```diff
@@ -36,7 +36,7 @@ resource "google_compute_instance" "dev-vm" {
   // ignore changes to current status and cpu_platform
   lifecycle {
     ignore_changes = [
-      machine_type,
+      machine_type
```

Comment:
https://github.com/hashicorp/terraform-provider-google/issues/13671

Annoying issue with max duration time being used with STOP instead of DELETE